python/interpret_text/introspective_rationale/introspective_rationale_explainer.py

```diff
@@ -47,10 +47,10 @@ def __init__(
         self.args = args
 
         if classifier_type == "BERT":
-            args.gen_embedding_dim = 768 # input dimension to use in the generator classifier
+            args.gen_embedding_dim = 768  # input dimension to use in the generator classifier
```

Comment:
the 768 would be a good thing to make a constant and reference it wherever it is used for maintainability